### DISCUSSION TITLE
Scaled 1080P50 NDI filter - 2160P50 project (canvas)

### DISCUSSION THREAD

**Discussion by sandervg** [DistroAV/DistroAV#1156] [Q&A - Support]
I am not sure if there is another workaround here but I would like to setup my OBS project in 2160P50 but have some scenes as dedicated ndi output (filters) to only be 1080P50. Using SDI out (decklink filter) there is an option to select the resolution. Would it be an idea to have this for NDI also? 
Have this option for the Main and Preview output would also come in handy to lower bandwidth usage.

**Comment by Trouffman** [comment]
Thanks for the suggestion. Would there be any other "use-cases" other than bandwidth reduction?

I can forsee that different resolution in filter could be a thing (with a resize filter prior to the NDI filter). but framerate would have to be the exact same (inherent Core logic in OBS)

**Comment by Trouffman** [comment]
You can do this with NDI output filter on a source that you set to the size you need.
Only resolution can be changed, not the framerate
